drivers/input/input_stmpe811.c

```diff
@@ -335,18 +335,17 @@ static void stmpe811_report_touch(const struct device *dev)
 	int x = data->touch_x;
 	int y = data->touch_y;
 
-	if (config->screen_width > 0 && config->screen_height > 0) {
-		x = (((int)data->touch_x - config->raw_x_min) * config->screen_width) /
+	if (config->common.screen_width > 0 && config->common.screen_height > 0) {
```

Comment:
you can set a `const struct input_touchscreen_common_config *common = &cfg->common;` as a local variable and use it here, make these a bit more compact